xkb/xkb.c

```diff
@@ -5899,6 +5899,10 @@ ProcXkbGetKbdByName(ClientPtr client)
             return BadAlloc;
         }
 
+        if (childbuf.wpos != (mrep.length * 4))
```

Comment:
Should it be `rpcbuf`? Or there are some other changes that aren’t here yet?

Reply:
whoops, those things may happen if one just blindly lets an automated script pick out commits from a longer queue :o 
originally this one came after another one that's renaming the local-scoped rpcbuf's to childbuf (because another one later, there's a func-global rpcbuf coming in).